trieste/models/model_interfaces.py

```diff
@@ -306,6 +321,14 @@ def sample(self, query_points: TensorType, num_samples: int) -> TensorType:
     def predict_y(self, query_points: TensorType) -> tuple[TensorType, TensorType]:
         return self.model.predict_y(query_points)
 
+    def get_kernel(self) -> gpflow.kernels.Kernel:
```

Comment:
Why is this method necessary? `predictor.model.kernel` seems quite concise, and highlights that the kernel is associated to the underlying model.

Reply:
I wanted to stress that only gpflow models have this attribute. More general (non-GP ones) will not

Reply:
Its just to inform people when they build new GPflow wrappers

Reply:
`predictor.model` has type `GPModel` for this class, and "not GP" models will not be able to call `.kernel` on the model. This should be highlighted by any editor which is aware of types, and will be caught by the static checks. I am not convinced of the need for another getter object on this class.